phoenix-protocol/src/main/PTable.proto

```diff
@@ -52,11 +52,12 @@ message PColumn {
 
 message PTableStats {
   required bytes key = 1;
-  repeated bytes values = 2;
+  optional bytes guidePosts = 2;
```

Comment:
yes it makes sense, as truncate of system.stats is necessary. I'll try this tomorrow.

Yes ,as per your previous comments , I'll be keeping "values" field at position 2, so this will automatically ensure that empty PGuidePosts is returned when client older than 4.7 is used right?
As per below code from version older than 4.7 (PTableImpl.createFromProto() )
GuidePostsInfo info =
                    new GuidePostsInfo(guidePostsByteCount, value, rowCount);//Prior 4.7 version :- empty "value" list.

3) I have created a GuidePostsInfoWriter and you can review the changes in this pull request now.